Firestore/core/src/api/aggregate_query.cc

```diff
@@ -42,11 +43,28 @@ AggregateQuery::AggregateQuery(Query query,
     : query_{std::move(query)}, aggregates_{std::move(aggregates)} {
 }
 
-void AggregateQuery::Get(AggregateQueryCallback&& callback) {
+void AggregateQuery::GetAggregate(AggregateQueryCallback&& callback) {
   query_.firestore()->client()->RunAggregateQuery(query_.query(), aggregates_,
                                                   std::move(callback));
 }
 
+// TODO(b/280805906) Remove this count specific API after the c++ SDK migrates
+// to the new Aggregate API
+void AggregateQuery::Get(CountQueryCallback&& callback) {
+  this->GetAggregate([callback](const StatusOr<ObjectValue>& result) {
```

Comment:
You should _move_ `callback` into the lambda. This is a cool new thing we can do now that we've "gotten with the times" and upgraded to c++14 ;)

Namely, something like this:

```cpp
this->GetAggregate([callback = std::move(callback)](const StatusOr<ObjectValue>& result) { ... }
```

Reply:
new and improved! Thanks